service/service.go

```diff
@@ -46,19 +50,50 @@ func New(conf *config.Config) (*Service, error) {
 		return nil, fmt.Errorf("failed to create a new gRPC client: %w", err)
 	}
 
-	return &Service{
-		Conf:          conf,
-		OraclePrivKey: oraclePrivKey,
-		UniqueID:      hex.EncodeToString(selfEnclaveInfo.UniqueID),
-		GrpcClient:    grpcClient.(*panacea.GrpcClient),
-	}, nil
+	subscriber, err := event.NewSubscriber(conf.Panacea.WSAddr)
+	if err != nil {
+		// TODO: close grpcClient
+		return nil, fmt.Errorf("failed to init subscriber: %w", err)
+	}
+
+	svc := &Service{
+		conf:          conf,
+		oracleAccount: oracleAccount,
+		oraclePrivKey: oraclePrivKey,
+		uniqueID:      hex.EncodeToString(selfEnclaveInfo.UniqueID),
+		grpcClient:    grpcClient.(*panacea.GrpcClient),
+		subscriber:    subscriber,
+	}
+
+	if err := svc.startSubscriptions(); err != nil {
+		return nil, fmt.Errorf("failed to start subscriptions: %w", err)
+	}
+
+	return svc, nil
 }
 
-func (s Service) Close() error {
+func (s *Service) startSubscriptions() error {
+	return s.subscriber.Run(
+		event.NewRegisterOracleEvent(s),
+	)
+}
```

Comment:
My guess is that this command may need to add a number of different Events in addition to the RegistratorOracleEvent.
I think it would be good to pass the Event as a parameter.
Hm... we'll have to think a bit more about how to put it in this structure.

Reply:
If this service is only run with one command (eg `doracled start`) then you can put all the events here.

Reply:
I'm not sure if I understood your comment correctly.
The `service.go` is only for the `doracled start` which runs a background process. And, the `subscriber.Run()` accepts multiple events as parameters. 

If we need to subscribe only one event in the different command such as `doracled hello..`, we can just create a `subscriber` separately without using the `Service`.

Reply:
Yes, when I said 'service' I meant 'service.go'.
As you said, if `service.go` is only used for `doraced start` then there is no problem.

What I was thinking was a concern when this `service.go` was used in other commands as well.

Reply:
I got you. The name 'service' is quite confusing in this doracle because doracle doesn't provide any service to clients. It just subscribe events in background. And, as you said, doracle can be started with many different modes.
Hmm. then, we should change the structure. Let me think about it. I'm not sure if we can find nice structure now before starting the next milestone.

Reply:
Let's close this PR until we can know which new commands will be added and how they will use a subscriber and a grpc client. At that time, we can refactor this nicely.